shared/teams/team/members/container.js

```diff
@@ -16,15 +16,26 @@ const getOrderedMemberArray = (
   you: ?string,
   listYouFirst: boolean
 ): Array<Types.MemberInfo> => {
-  let youInfo
   let info = memberInfo
-  if (you && !listYouFirst) {
-    youInfo = memberInfo.find(member => member.username === you)
-    if (youInfo) {
-      info = memberInfo.delete(youInfo)
-    }
-  }
   let returnArray = info.toArray().sort((a, b) => {
+    if (!a.active) {
+      if (!b.active) {
+        // both are inactive, compare usernames
+        return a.username.localeCompare(b.username)
+      }
+      // b is active, should go later
+      return -1
+    } else if (!b.active) {
+      // b is inactive, should come first
+      return 1
+    }
+    if (listYouFirst && you) {
```

Comment:
doesn't this listYouFirst logic trump everything? aka it should be above this active checking stuff

Reply:
The ticket shows it this way in Zeppelin (reset user before self)

Reply:
also, per the design in the ticket, I added the chat button back in on the cancelled row